### PR TITLE
 Added Camera Datamodel

### PR DESCRIPTION
**Pull request by shyam28598**

### CODE REVIEW

File: dataModel.Camera/examples/example.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	"streamURL": "https://drive.google.com/file/d/1eNmgWDvb2R34o03cZ9dPXrtEvsreQzQ4/view?usp=sharing",
+	"cameraUsage": "SURVEILLANCE",
+	"cameraType": "FIXED",
+	"endDateTime": "2021-05-11T06:35:20.065Z",
+    "startDateTime": "2021-05-11T06:30:00.020Z",
+	"cameraOrientation": {
+        "comments":"Camera facing RSBhawan",
+        "annotatedMap":"https://drive.google.com/file/d/1RXDGogU5UMmzRppqFaNKTzcr69Kl0wjb/view?usp=sharing"
+    },
+	"location": {
```
Comment (albertoabellagarcia) [inline dataModel.Camera/examples/example.jsonld:15]: missing "type": "Point"

File: dataModel.Camera/schema.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                "streamName": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Name of the video stream from the camera corresponding to this observation"
+                },
+                "startDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Property. Model:'https://schema.org/DateTime'. Reported start time corresponding to this observation."
+                },
+                "location": {
```
Comment (albertoabellagarcia) [inline dataModel.Camera/schema.json]: Location is already included with the seccion location-commons, you are duplicating, please remove.

File: dataModel.Camera/examples/example.jsonld [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+	"@context": ["https://schema.lab.fiware.org/ld/context",
```
Comment (albertoabellagarcia) [inline dataModel.Camera/examples/example.jsonld]: this context is old, better to link https://smartdatamodels.org/context.jsonld

**Review by albertoabellagarcia — COMMENTED** [review]
Please add also the ADOPTERS.yaml 
https://github.com/smart-data-models/data-models/blob/master/templates/dataModel/ADOPTERS.yaml
and if you want to be credited the CONTRIBUTORS.yaml
https://github.com/smart-data-models/data-models/blob/master/templates/Subject/CONTRIBUTORS.yaml
for customization purposes you can also use the notes.yaml
See an example
https://github.com/smart-data-models/dataModel.Aeronautics/blob/master/Aircraft/notes.yaml

File: dataModel.Camera/schema.json (hunk 2 above)
Comment (shyam28598) [reply dataModel.Camera/schema.json]: > Please add also the ADOPTERS.yaml
> https://github.com/smart-data-models/data-models/blob/master/templates/dataModel/ADOPTERS.yaml
> and if you want to be credited the CONTRIBUTORS.yaml
> https://github.com/smart-data-models/data-models/blob/master/templates/Subject/CONTRIBUTORS.yaml
> for customization purposes you can also use the notes.yaml
> See an example
> https://github.com/smart-data-models/dataModel.Aeronautics/blob/master/Aircraft/notes.yaml

Hey the ADOPTERS.yaml  file is a little confusing, Is it mandatory to include that file?, Other changes have been made. I will raise another PR  with the new changes

**Review by albertoabellagarcia — APPROVED** [review]

**Comment by albertoabellagarcia** [comment]
Approved, the data model has been provisionally published in the subject [dataModel.Device](https://github.com/smart-data-models/dataModel.Device/tree/master/Camera), please complete the [Contribution Agreement](https://smartdatamodels.org/index.php/contributor-agreement/) and if you want to be credited please PR on this file [CONTRIBUTORS.yaml](https://github.com/smart-data-models/dataModel.Device/blob/master/CONTRIBUTORS.yaml)

**Pull request closed without merging**